### ISSUE TITLE
AAC Is Kicking And Broadcasting And I'm Sure I Set It Not To? How Do I Remove Them?

### ISSUE THREAD

**Issue by MachoPiggies** [konsolas/AAC-Issues#923]
### Issue description
Please include a detailed description of the issue, instructions to reproduce it, client names/links (if applicable), and a relevant paste from plugins/AAC/logs. Please include as much information as possible, including anything about the environment that may be relevant (e.g. screenshot of the location). A video with (config.yml:verbose -> true) is useful but not required. It should not be substitued for a description.

If you a reporting a console error, take the earliest version in the console log to make sure none of the error is cut off.

Large blocks of text should be pasted on https://gist.github.com/ or within markdown code blocks. 

### Core information
**Server version**: Spigot 1.11.2 -R0.1-SNAPSHOT

**AAC version**: 3.3.1

**ProtocolLib version**: 4.3.0

**Plugin list**: http://prntscr.com/ggvv11

### Special environment information
**ViaVersion version**: 1.2.0

**ProtocolSupport version**: N/A

**ViaBackwards version**: N/A

**BungeeCord plugins**: N/A

### Configuration
Please describe any modifications you have made to the default config. You do not need to include information about the "thresholds" section of any check, unless you are reporting a false kick/ban. 

**General Config**
`#### AAC Config
#
# NOTE: Most of the values here have been tried and tested to work.
#       It is not recommended that you change them unless you are
#       getting false positives.
#		
#		It would be a good idea to make sure the false positives you're getting aren't actually hackers....
#
#
# The configurable commands will replace "{player}" with the name of the player.
# You can run multiple commands on a single threshold by creating 2 entries with the same key:
#  thresholds:
#     10:
#      - "broadcast {player} was kicked for hacking"
#      - "kick {player} hacking"
#
# Be sure to check language.yml for configuration regarding staff chat, etc.
#
## If you need any help configuring AAC, or you're getting false positives and you don't know why, feel free to PM me on Spigot.

# Angle forces players to look at entities they want to attack
angle:
   # Do you want to enable the angle check
   enabled: true
   
   # How far off the entity should the player be able to look
   max_diff: 0.30

   # How far off a player should the attacker be able to look?
   max_player_diff: 0.15
   
   # How close must the players be to bypass the angle check?
   min_dist: 2.0
   
   # Should AAC cancel false hits
   cancel_false_hits: true
   
   # Map of violation levels to commands
   thresholds:
      6: "aacstaffnotify MINOR: {player} is suspected for Killaura (Angle) (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MODERATE: {player} is suspected for Killaura (Angle) (ping = {ping}, tps = {tps})"
      18: "aacstaffnotify MAJOR: {player} is suspected for Killaura (Angle) (ping = {ping}, tps = {tps})"

# BadPackets prevents players from using packets to exploit the server
badpackets:
   # Do you want to enable the badpackets check
   enabled: true
   
   # After how many moves should packets be cancelled?
   cancel_threshold: 14
   
   # How many moves should a player be able to send in 3 ticks
   # An unmodified client should send 3 moves in 3 ticks. Be sure to think about lag
   # The violation value will be increased by (how many packets the player sends - max_moves) / 10
   max_moves: 50
   
   # Map of violation levels to commands
   thresholds:
      30: "aacstaffnotify MINOR: {player} is suspected for Regen/NoPotion (ping = {ping}, tps = {tps})"
      60: "aacstaffnotify MODERATE: {player} is suspected for Regen/NoPotion (ping = {ping}, tps = {tps})"
      90: "aacstaffnotify MAJOR: {player} is suspected for Regen/NoPotion (ping = {ping}, tps = {tps})"
   
# Criticals prevents players from getting critical hits when it's not possible
criticals:
   # Do you want to enable the criticals check?
   enabled: true
   
   # Should a hit be cancelled if it is a critical and the player is on the ground?
   cancel_false_hit: true
   
   # Map of violation levels to commands
   thresholds:
      1: "aacstaffnotify MINOR: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"
      3: "aacstaffnotify MODERATE: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"
      5: "aacstaffnotify MAJOR: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"

# Fastbow prevents players from shooting their bow too quickly
fastbow:
   # Do you want to enable the fastbow check
   enabled: true
   
   # What is the minimum force of a shot for it to be registered
   # This goes from 0 to 1, 1 being a fully charged bow
   min_force: 0.15
   
   # How many of these shots should a player be able to fire in 1 second
   # it takes 1.2 seconds to fully draw a bow
   shots_fired_threshold: 3
   
   # Map of violation levels to commands
   thresholds:
      3: "aacstaffnotify MINOR: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"
      6: "aacstaffnotify MODERATE: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"
      9: "aacstaffnotify MAJOR: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"

# FightSpeed prevents players from attacking too quickly
fightspeed:
   # Do you want to enable the fightspeed check
   enabled: true
   
   # How many times should a player be able to attack in 1 second
   # NoCheatPlus's fightspeed check allows 15 hits per second
   max_hits_per_second: 30
   
   # Map of violation levels to commands
   thresholds:
      15: "aacstaffnotify MINOR: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"

# NormalMovements stops players from moving unexpectedly
normalmovements:
   # Do you want to enable the normalmovements check
   enabled: true

   # f:jump:x - Should NormalMovements handle jumping?
   f_jump: true

   # f:step:x - Should NormalMovements handle stepping?
   f_step: true

   # f:float:x - Should NormalMovements handle floating?
   f_float: true

   # f:dy:x - Should NormalMovements attempt to detect client patterns?
   f_dy: true

   # Map of violation levels to commands
   thresholds:
      20: "aacstaffnotify MINOR: {player} is suspected of movement hacking (ping = {ping}, tps = {tps})"
      40: "aacstaffnotify MODERATE: {player} is suspected of movement hacking (ping = {ping}, tps = {tps})"
      50: "aacstaffnotify MAJOR: {player} is suspected of movement hacking (ping = {ping}, tps = {tps})"

# Fly kicks players who move too high
fly:
   # Do you want to enable the fly check
   enabled: true
   
   # Base max jump height. Expected max is 1.3
   max_jump: 1.42

   # If a player's violation level is below 2, he will not be rubberbanded if the violation amount is not more than allow.
   allow: 0.07
   
   # How high a player will be allowed to jump after bouncing off a slime block
   slime_allow: 60.0

   # Increasing velocity_multiplier will improve compatiblility with jump pads.
   velocity_multiplier: 2.4
   
   # How high should a player be allowed to jump after taking damage
   knockback_allow: 3.1
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for Fly (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for Fly (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR: {player} is suspected for Fly (ping = {ping}, tps = {tps})"
   
# Speed kicks players who move too quickly
speed:
   # Do you want to enable the speed check
   enabled: true

   # Speed limits per tick
   sprinting_speed: 0.41
   sneaking_speed: 0.21

   # On and under water
   surface_speed: 0.23
   underwater_speed: 0.19
   dive_speed: 0.69

   # Other
   cobweb_speed: 0.10
   item_speed: 0.21
   soul_sand_speed: 0.39
   ice_speed: 0.71

   # Multiplier of horizontal velocity for movement
   horizontal_velocity: 4.2

   # Required horizontal velcoity for a midair check
   min_horizontal_velocity: 0.1

   # Multiplier of the maximum distance if the player is jumping
   jump_multiplier: 1.7

   # Ticks a player should not be checked after entering the water
   dive_period: 20

   # Should AAC use movement counting to prevent clients from bypassing by sending too many moves?
   use_move_counting: true

   # How many moves per second should be assumed as legitimate? (NCP = 22)
   max_moves: 22

   # Custom modifiers for anti-timer, averaging over a longer period of time
   custom:
      # Second line of defense over timer. Slow, but practically unbypassable.
      enabled: true
      # Max moves per second
      max_moves: 21.0
      # Ticks to take an average over
      time: 150
      # Whether this rubberbanding should ignore the vls_to_band anti false positive limitations
      bypass_band: true

   # How many times should a player be allowed to toggle sneak in a second (flare hacked client)
   max_sneak_toggles: 15

   # How many times should a player be allowed to toggle use in a second (flare hacked client)
   max_use_toggles: 15

   # Multiplier for speed. Increase if you have false positives.
   speed_multiplier: 0.32

   # Multiplier for slowness. Decrease if you have false positives.
   slow_multiplier: 0.03

   # Every time a player fails the speed check, the band vl will go up by (actualDistance - expectedDistance) * band_multiplier
   # This can be set to 0 to turn off rubberbanding completely.
   band_multiplier: 5.0

   # How many violations before a player will start being rubber-banded
   vls_to_band: 8

   # How long must a player remain clean for band violations to be cleared? (seconds)
   clear_vl_delay: 12

   # Map of violation levels to commands
   thresholds:
      8: "aacmessage {player} It looks like you might be glitching, please tap sneak and block with your sword"
      8: "aacstaffnotify MINOR: {player} is suspected for Speed (ping = {ping}, tps = {tps})"
      16: "aacstaffnotify MODERATE: {player} is suspected for Speed (ping = {ping}, tps = {tps})"
      24: "aacstaffnotify MAJOR: {player} is suspected for Speed (ping = {ping}, tps = {tps})"

# HeadRoll stops players being derps :/
headroll:
   # Do you want to enable the headroll check
   enabled: true

   # Map of violation levels to commands
   thresholds:
      1: "aacstaffnotify MINOR: {player} is suspected for Derp (ping = {ping}, tps = {tps})"
      3: "aacstaffnotify MODERATE: {player} is suspected for Derp (ping = {ping}, tps = {tps})"
      5: "aacstaffnotify MAJOR: {player} is suspected for Derp (ping = {ping}, tps = {tps})"

# Forcefield detects combat hacks which hit entities around the player
forcefield:
   # Do you want to enable the forcefield check
   enabled: true
   
   # Do you want the player above you to be invisible
   # *HIGHLY* recommended to be kept at false.
   # The majority of hacked clients do not target invisible entities.
   # If you change this to true, killaura detection will be severely slowed down. Best idea would be to ask your players to ignore it.
   head_player_invisible: false

   # You can make the head player invisible when the player isn't fighting
   # With this true, the head player will be invisible when the player isn't fighting
   # If the player isn't fighting or this option is false, AAC will default to head_player_invisible
   # This takes PVP from combat_time, so make sure that's a sensible value
   invisible_outside_pvp: true
   
   # How high above the player should the headplayer be spawned?
   # Anything under 2.7 could affect hit detection
   # Anything above 4.5 could be out of range of the player
   head_player_height: 3.6
   
   # Should the head player be moved away if the player looks up?
   # This is to prevent players from creating false positives by hitting the head player (if it is visible)
   no_head_player_looking_up: true
   
   # Do you want to spawn the side players
   # This spawns players within the users' FOV to confuse click aimbot
   # Side players are invisible, as they are within the user's FOV.
   use_side_players: true
   
   # Use player names and gameprofiles from past players instead of random profiles
   # This can be extremely useful in preventing bypasses, as the player's details are completely legitimate
   # This setting has no effect if online_player_names is true.
   past_player_names: false

   # Should the head player use profiles of players online right now?
   # If you have nametag plugins that use the scoreboard, keep this disabled
   # Otherwise, turn this on as it patches a massive number of possible bypasses
   online_player_names: false

   # Do you want to keep the head player in the tab list?
   # This can prevent bypasses where the the client checks that the name is in the tablist, before hitting it.
   head_tablist: false
   
   # Map of violation levels to commands
   thresholds:
      7: "aacstaffnotify MINOR: {player} is suspected for Killaura (NPC check) (ping = {ping}, tps = {tps})"
      14: "aacstaffnotify MODERATE: {player} is suspected for Killaura (NPC check) (ping = {ping}, tps = {tps})"
      21: "aacstaffnotify MAJOR: {player} is suspected for Killaura (NPC check) (ping = {ping}, tps = {tps})"
   
# Knockback checks if the player moves after he is hit
knockback:
   # Do you want to enable the knockback check
   enabled: true
   
   # How long should AAC wait before checking a player
   tick_delay: 9

   # Should AAC compensate for ping?
   add_ping: true

   # What's the maximum ping value AAC should compensate to (pingspoof?)
   max_ping: 500

   # What's the minimum distance a player should be knocked back
   min_knockback_dist: 0.01

   # Minimum time a player should be knocked back for
   min_knockback_time: 2
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for Anti-Knockback (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for Anti-Knockback (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for Anti-Knockback (ping = {ping}, tps = {tps})"

# Reach checks if entites the player hits are close enough to the player
reach:
   # Do you want to enable the reach check
   enabled: true
   
   # How far should AAC allow the player to reach (vanilla = 3.8)
   max_reach: 4.1
   
   # How far does bukkit allow players to reach (Don't change this)
   max_bukkit_reach: 9.0

   # How much should AAC compensate for hitbox stretching? Increase this first with false positives.
   move_multiplier: 3.0
   
   # Map of violation levels to commands
   thresholds:
      15: "aacstaffnotify MINOR: {player} is suspected for Reach (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for Reach (ping = {ping}, tps = {tps})"
      28: "aacstaffnotify MAJOR: {player} is suspected for Reach (ping = {ping}, tps = {tps})"

# NoSwing makes sure players swing their arm when attacking
noswing:
   # Do you want to enable the noswing check
   enabled: true

   # Map of violation levels to commands
   thresholds:
      8: "aacstaffnotify MINOR: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"
      16: "aacstaffnotify MODERATE: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"
      22: "aacstaffnotify MAJOR: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"

# Regen directly stops players from healing too quickly
regen:
   # Do you want to enable the regen check
   enabled: true

   # Should AAC cancel regeneration if it happens too quickly?
   cancel_regen: true

   # What delay should AAC force between regeneration?
   min_regen_delay: 3600

   # Map of violation levels to commands
   thresholds:
      8: "aacstaffnotify MINOR: {player} is suspected for Regen (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MODERATE: {player} is suspected for Regen (ping = {ping}, tps = {tps})"
      16: "aacstaffnotify MAJOR: {player} is suspected for Regen (ping = {ping}, tps = {tps})"

# Killaura detects combat hacks which give an unfair advantage
killaura:
   # Do you want to enable the killaura check
   enabled: true

   # Killaura also has a different method of using NPCs. Check_chance to 0 to disable.
   npc:
      # What % of the time should AAC do an NPC check?
      check_chance: 100

      # How long should AAC wait to spawn the NPC
      check_delay: 5

      # Should AAC use a random entity type for the NPC? false = NPC will be the type of entity the player is fighting with.
      random_type: true

      # After spawning the NPC, how long should AAC wait before starting to count hits?
      # check_delay + listen_delay must be less than 9, or hits will be lost.
      listen_delay: 4

      # How far away must 2 players be to be checked?
      min_dist: 2.0

      # How long should the NPC be spawned for?
      ticks_spawned: 9

      # How far along the distance should the NPC be spawned
      percent_distance: 80

      # How much should the NPC check be weighted per hit?
      npc_multiplier: 3

      # How far off should a player be able to look to allow the hit
      max_diff: 0.11

      # How far should the player be allowed to reach through the NPC
      max_reach: 4.5

      # How long should a player be prevented from hitting someone when he fails this check? (ticks)
      suspended_combat_time: 6

   # Heuristic components: These will instantly add 50 to a player's violation level when the player has a high likelihood of hacking.

   # Requried chance for an Aimbot kick (%). Set to something >= 100 to disable
   aimbot: 80

   # Required chance for a Reach kick (%). Set to something >= 100 to disable
   # EXPERIMENTAL
   reach: 89

   # Map of violation levels to commands
   thresholds:
      9: "aacstaffnotify MINOR: {player} is suspected for Killaura (Heuristics) (ping = {ping}, tps = {tps})"
      14: "aacstaffnotify MODERATE: {player} is suspected for Killaura (Heuristics) (ping = {ping}, tps = {tps})"
      17: "aacstaffnotify MAJOR: {player} is suspected for Killaura (Heuristics) (ping = {ping}, tps = {tps})"

# CombatImpossible cancels hits which cannot happen normally
combatimpossible:
   # Do you want to enable the CombatImpossible check
   enabled: true

   # Should hits be cancelled if the attacker is dead?
   cancel_dead: true

   # Should hits be cancelled if the attacker is using an item?
   cancel_using_item: true

   # Map of violation levels to commands
   thresholds:
      12: "aacstaffnotify MINOR: {player} appears to be hitting illegitimately (ping = {ping}, tps = {tps})"
      18: "aacstaffnotify MODERATE: {player} appears to be hitting illegitimately (ping = {ping}, tps = {tps})"
      24: "aacstaffnotify MAJOR: {player} appears to be hitting illegitimately (ping = {ping}, tps = {tps})"

# NoFall ensures players take fall damage
nofall:
   # Do you want to enable the NoFall check
   enabled: true

   # Minimum downwards velocity for NoFall to apply
   required_velocity: -0.2

   # Should AAC deal fall damage to players?
   deal_damage: true
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for NoFall (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for NoFall (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for NoFall (ping = {ping}, tps = {tps})"

# Glide ensures players fall at the right speed
glide:
   # Do you want to enable the Glide check
   enabled: true
   
   # How strongly should AAC enforce fall speeds (smaller values are more accurate)
   tolerance: 0.7

   # Teleport the player down if detected?
   teleport: false
   
   # Map of violation levels to commands
   thresholds:
      25: "aacstaffnotify MINOR: {player} is suspected for Glide (ping = {ping}, tps = {tps})"
      45: "aacstaffnotify MODERATE: {player} is suspected for Glide (ping = {ping}, tps = {tps})"
      55: "aacstaffnotify MAJOR: {player} is suspected for Glide (ping = {ping}, tps = {tps})"

# Climb stops people climbing ladders too quickly
climb:
   # Do you want to enable the Climb check
   enabled: true

   # How fast should a player be allowed to start a ladder climb
   max_start_speed: 1.5

   # How fast should a player be allowed to climb a ladder
   max_ascent_speed: 0.38

   # Map of violation levels to commands
   thresholds:
      15: "aacstaffnotify MINOR: {player} is suspected for FastLadder (ping = {ping}, tps = {tps})"
      17: "aacstaffnotify MODERATE: {player} is suspected for FastLadder (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastLadder (ping = {ping}, tps = {tps})"

# Jesus stops people from walking on water
jesus:
   # Do you want to enable the Jesus check
   enabled: true

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"

# Phase stops people glitching through blocks
phase:
   # Do you want to enable the Phase check
   enabled: true

   # How long should a player be frozen in place after being detected for phase? (ms)
   freeze_time: 200

   # What's the maximum distance AAC should teleport players to ideal locations.
   max_dist: 3.0

   # Should AAC check for vclip
   check_vclip: true

   # Map of violation levels to commands
   thresholds:
      7: "aacstaffnotify MINOR: {player} is suspected for Phase (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for Phase (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for Phase (ping = {ping}, tps = {tps})"

# FastUse stops players consuming items too quickly
fastuse:
   # Do you want to enable the FastUse check
   enabled: true

   # What delay should AAC enforce for eating an item (ms)
   ms_to_use: 1300

   # Map of violation levels to commands
   thresholds:
      5: "aacstaffnotify MINOR: {player} is suspected for FastEat (ping = {ping}, tps = {tps})"
      10: "aacstaffnotify MODERATE: {player} is suspected for FastEat (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MAJOR: {player} is suspected for FastEat (ping = {ping}, tps = {tps})"

# FastBreak stops players from breaking blocks too quickly
fastbreak:
   # Do you want to enable the fastbreak check?
   enabled: true

   # How many ticks should fastbreak count as lag, and ignore
   let_ticks: 2

   # Should AAC cancel an illegal block break?
   cancel_break: true

   # Should AAC check for a break delay (i.e. between block breaks)
   check_delay: true

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"

# FastPlace stops players from placing blocks too quickly
fastplace:
   # Do you want to enable the fastplace check?
   enabled: true

   # How many ms should fastplace force between block placements?
   min_delay: 100

   # Should AAC cancel an illegal block place?
   cancel_place: true

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"

# Nuker stops players from breaking too many blocks at once
nuker:
   # Do you want to enable the nuker check?
   enabled: true

   # How many blocks should a player be able to break in 1 second: vl if greater than this value
   max_breaks_per_second: 40

   # The violation will only be recorded if it divides by this number (reduces spam)
   vl_divisor: 30

   # Map of violation levels to commands
   thresholds:
      5: "aacstaffnotify MINOR: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"
      8: "aacstaffnotify MODERATE: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MAJOR: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"

# ImpossibleInteract stops players interacting with blocks they can't
impossibleinteract:
   # Do you want to enable the impossibleinteract check
   enabled: true

   # Detect interacting with liquids?
   cancel_liquids: true

   # Detect ghosthands?
   cancel_ghosthand: true

   # Map of violation levels to commands
   thresholds:
      14: "aacstaffnotify MINOR: {player} is interacting suspiciously (ping = {ping}, tps = {tps})"
      25: "aacstaffnotify MODERATE: {player} is interacting suspiciously  (ping = {ping}, tps = {tps})"

# PlayerRadar hides players that cannot be seen
playerradar:
   # Do you want to enable anti-radar? (Experimental)
   enabled: false

   # How far must 2 players be apart to be hidden from each other?
   # Players closer than this will always be able to see each other
   # In games where nametags are not shown, this can be set to 0.
   required_dist: 15

   # This should be set to your entity render distance (Spigot default = 48)
   max_dist: 48

   # Map of violation levels to commands
   thresholds:
      0: "Unused"

# Spam prevents players from spamming the server chat
spam:
   # Do you want to enable the spam check?
   enabled: true
   
   # What words should not be covered by the anti-spam check?
   # messages less than 3 letters are already ignored
   whitelist:
     - "lol"
     - "hello"
     - "okay"
   
   # How many messages should a player be able to send in 10 seconds
   message_limit: 4
   
   # How fast can a player send messages (ticks)
   chat_speed: 10
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify {player} is probably spamming"

### General things

# How many times must a player be kicked before he is banned (-1 for no autoban)
ban_threshold: 10

# What command should be executed when a player is banned (only applies for /aackick)
ban_command: "ban {player}"

# How long after a player is kicked should he be allowed to log in (ticks)
kick_ticks: 120

# How long before a player leaves combat after dealing/taking damage
combat_time: 50

# Should AAC kill combat loggers (combat_time)
kill_loggers: true

# Should AAC log events to a file?
log_file: true

# Should AAC log events to the console?
log_console: true

# Should /aackick broadcast a message?
aackick_broadcast: false

# Should AAC disable/enable checks dynamically? IF YOU HAVE PWP DISABLE
usetps: true

# Disable the command /aac? If you really want to ;(
aac: true

# Enable verbose logging? This will allow those with the AAC.verbose permission to get spammed.
# Not recommended.
verbose: false
`

**Language Config**
`# AAC messages configuration file

# Prefix (wooooo yeah)
prefix: "&a&k::&r&6&lIAC&a&k::&r : &6"

# Spam messages
spam_too_fast: "You can't chat that fast!"
spam_slow_chat: "Please slow down chat"
spam_repeat: "Please refrain from repeating yourself"
spam_repeat_other: "You can't repeat that"

# Command messages
aac_kick_broadcast: ""
ban_message: "#anticheat"

relog_message: "Wait a few seconds before logging back in."

# Fastbow
too_many_shots: "You're firing shots too quickly. Please wait 5 seconds before using your bow again."
allow_bow: "You may now use your bow again"

# Staff chat
prefix_to_speak: "@"
# This will also be used for the notifications from aacstaffnotify in the config
format: "&a&k::&r&6&lIAC&a&k::&r : &6 &2[{sender}] &r"

# Combat logging
killed_for_logging: "You were killed for combat logging. Don't do it."
logging_broadcast: "{player} was killed for combat logging. Don't do it."
`

For the lazy, you may use this [online diff tool](https://www.diffchecker.com/) - make sure you select "Store forever".

**Comment by CeIebrimbor** [comment]
You are using a config from 2016. Make sure you get the latest AAC and regenerate your config

**Comment by MachoPiggies** [comment]
After updating, it has become much worse.

- Teleports all my donors and staff to the floor when they are flying
- Still sends kick messages, http://prntscr.com/ghgyfm, step isn't even in the config.

Please help.

**Comment by CeIebrimbor** [comment]
- Where did you download AAC? 
- What version is running when you run the /aac command? 
- What is your current config.yml, seeing as how your config should be completely different from the one you posted originally (assuming you actually updated to latest).

**Comment by MachoPiggies** [comment]
Main Spigot Site

3.3.1

`#### AAC Config (Generated by AAC 3.3.1)
#
# NOTE: Most of the values here have been tried and tested to work.
#       It is not recommended that you change them unless you are
#       getting false positives.
#
#       It would be a good idea to make sure the false positives you're getting aren't actually hackers....
#
#
# The configurable commands will replace "{player}" with the name of the player.
# You can run multiple commands on a single threshold by creating 2 entries with the same key:
#  thresholds:
#     10:
#      - "broadcast {player} was kicked for hacking"
#      - "kick {player} hacking"
#
# Be sure to check language.yml for configuration regarding staff chat, etc.

# Hitbox forces players to look at entities they want to attack.
hitbox:
   # Do you want to enable the angle check
   enabled: true

   # How much should AAC expand hitboxes by?
   lenience: 0.6

   # Should AAC cancel illegal hits?
   cancel: true
   
   # How far should a player be able to reach?
   # This value is added to the target entity hitbox size before being checked.
   max_reach: 3.5

   # How much should AAC compensate for the movement of the target?
   # This only affects reach, not hitbox lenience.
   move_compensation: 2.0

   # Minimum number of violations before AAC starts cancelling hits
   min_vl: 1
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player}  is suspected for Killaura (Hitbox) (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for Killaura (Hitbox) (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR: {player} is suspected for Killaura (Hitbox) (ping = {ping}, tps = {tps})"

# BadPackets prevents players from using packets to exploit the server
badpackets:
   # Do you want to enable the badpackets check
   enabled: true
   
   # After how many moves should packets be cancelled?
   # A lower value reduces the possibility of blinking and exploits and improves performance.
   # A higher value will reduce the number of false positives. This should always be above 3.
   cancel_threshold: 30
   
   # How many moves should a player be able to send in 3 ticks
   # An unmodified client should send 3 moves in 3 ticks. Be sure to think about lag
   # The violation value will be increased by (how many packets the player sends - max_moves) / 2
   max_moves: 10
   
   # Map of violation levels to commands
   thresholds:
      100: "aacstaffnotify MINOR: {player} is suspected for BadPackets (Probably Timer) (ping = {ping}, tps = {tps})"
      150: "aacstaffnotify MODERATE: {player} is suspected for BadPackets (Probably Timer) (ping = {ping}, tps = {tps})"
      200: "aacstaffnotify MAJOR: {player} is suspected for BadPackets (Probably Timer) (ping = {ping}, tps = {tps})"

# Criticals prevents players from getting critical hits when it's not possible
criticals:
   # Do you want to enable the criticals check?
   enabled: true
   
   # Should a hit be cancelled if it is a critical and the player is on the ground?
   cancel_false_hit: true

   # How many ticks should AAC enforce between jumps
   jump_delay: 5
   
   # Map of violation levels to commands
   thresholds:
      3: "aacstaffnotify MINOR: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"
      6: "aacstaffnotify MODERATE: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"
      10: "aacstaffnotify MAJOR: {player} is suspected for Criticals (ping = {ping}, tps = {tps})"

# Fastbow prevents players from shooting their bow too quickly
fastbow:
   # Do you want to enable the fastbow check
   enabled: true
   
   # How much lag compensation should AAC perform when checking for fastbow?
   lenience_ms: 130

   # How long should AAC block bow usage after a player is detected for fastbow? (set to 0 to disable)
   block_ticks: 100

   # How much VL does a player need before AAC starts blocking shots?
   min_vl: 5
   
   # Map of violation levels to commands
   thresholds:
      3: "aacstaffnotify MINOR: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"
      10: "aacstaffnotify MODERATE: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastBow (ping = {ping}, tps = {tps})"

# FightSpeed prevents players from attacking too quickly
fightspeed:
   # Do you want to enable the fightspeed check
   enabled: true
   
   # How many times should a player be able to attack per second (successful hits)
   max_hit: 15

   # How many times should a player be allowed to swing per second (CPS)
   max_swing: 19

   # Should fightspeed check for consistent click speed
   consistent_check: true

   # Maximum range of hit delays to be regarded as consistent
   consistent_threshold: 25
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR: {player} is suspected for AutoClicker (FightSpeed) (ping = {ping}, tps = {tps})"

# Fly prevents players in survival mode from flying
fly:
   # Do you want to enable the fly check
   enabled: true

   # How strongly should AAC enforce fall speeds (smaller values are more accurate)
   tolerance: 0.015
   e_tolerance: 0.2

   # Multiplier of violation size to be added to VL level
   vl_multiplier: 6.0

   # Minimum violation level for a player to be teleported down. Only if teleport is true
   min_vl: 100

   # Ticks after velocity a player should be immune to the glide check
   vel_ticks: 30

   # Teleport the player down if detected?
   teleport: false

   # Check for invalid abilities packets?
   abilities: true

   # How many ticks should AAC wait before reducing a player's violation level
   reduce_delay: 20

   # Should AAC reset a player's pitch to 0 after a detection?
   reset_pitch: false
   
   # Map of violation levels to commands
   thresholds:
      100: "aacstaffnotify MINOR: {player} is suspected for Flight (ping = {ping}, tps = {tps})"
      200: "aacstaffnotify MODERATE: {player} is suspected for Flight (ping = {ping}, tps = {tps})"
      300: "aacstaffnotify MAJOR: {player} is suspected for Flight (ping = {ping}, tps = {tps})"

# Speed prevents players in survival mode from moving too quickly
speed:
   # Do you want to enable the speed check
   enabled: true

   # Maximum amount (in blocks) a player should be allowed to go over a calculated limit.
   threshold: 0.005

   # Duration of time AAC should compensate for velocity (ticks). This is basically lag compensation.
   vel_ticks: 30

   # Multiplier of violation size to be added to VL level
   vl_multiplier: 6.0

   # How many times should a player be allowed to toggle sneak in a second
   max_sneak_toggles: 15

   # How many times should a player be allowed to toggle use in a second
   max_use_toggles: 35

   # How many violations before a player will start being teleported
   min_vl: 36

   # How many ticks should AAC wait before reducing a player's violation level
   reduce_delay: 20

   # Specific configuration options. Don't touch these unless you know what you're doing.
   #item_scalar: 0.5
   #sneak_scalar: 0.63

   # Map of violation levels to commands
   thresholds:
      50: "aacstaffnotify MINOR: {player} is suspected for Speed (ping = {ping}, tps = {tps})"
      100: "aacstaffnotify MODERATE: {player} is suspected for Speed (FightSpeed) (ping = {ping}, tps = {tps})"
      150: "aacstaffnotify MAJOR: {player} is suspected for Speed (FightSpeed) (ping = {ping}, tps = {tps})"

# Derp stops some forms of derp hacking.
derp:
   # Do you want to enable the derp check
   enabled: true

   # Map of violation levels to commands
   thresholds:
      1: "aacstaffnotify MINOR: {player} is suspected for Derp. Please investigate just incase. (ping = {ping}, tps = {tps})"

# Knockback checks if the player moves after he is hit
# Obsolete, see the velocity check
knockback:
   # Do you want to enable the knockback check
   enabled: false
   
   # legacy configuration
   tick_delay: 9
   add_ping: true
   max_ping: 500
   min_knockback_dist: 0.01
   min_knockback_time: 2

   thresholds:
     10: "aacstaffnotify MINOR: {player} is suspected for AntiKnockback (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for AntiKnockback (FightSpeed) (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for AntiKnockback (FightSpeed) (ping = {ping}, tps = {tps})"

# NoSwing makes sure players swing their arm when attacking
noswing:
   # Do you want to enable the noswing check
   enabled: true

   # Map of violation levels to commands
   thresholds:
      20: "aacstaffnotify MINOR: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"
      40: "aacstaffnotify MODERATE: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"
      60: "aacstaffnotify MAJOR: {player} is suspected for NoSwing (ping = {ping}, tps = {tps})"

# Regen directly stops players from healing too quickly
regen:
   # Do you want to enable the regen check
   enabled: true

   # Should AAC cancel regeneration if it happens too quickly?
   cancel_regen: true

   # What delay should AAC force between regeneration?
   min_regen_delay: 3600

   # What delay should AAC force if the player is fully satiated
   min_satiated_delay: 450

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for Regen (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for Regen (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR: {player} is suspected for Regen (ping = {ping}, tps = {tps})"

# Killaura detects combat hacks which give an unfair advantage
killaura:
   # Do you want to enable the killaura check
   enabled: true

   # What % of the time should AAC do a check?
   check_chance: 100

   # Lag compensation paramters - try not to break
   check_delay: 3
   listen_delay: 6

   # How far away must 2 players be to be checked?
   min_dist: 4

   # How long should the check last?
   duration: 60

   # How much should the check be weighted?
   weight: 6

   # How far off should a player be able to look to allow the hit
   max_diff: 0.11

   # How far should the player be allowed to reach during the check
   max_reach: 4.5

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for KillAura (NPC) (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for KillAura (NPC) (ping = {ping}, tps = {tps})"
      40: "aacstaffnotify MAJOR: {player} is suspected for KillAura (NPC) (ping = {ping}, tps = {tps})"

# Heuristics are used by AAC to detect advanced killaura hacks that bypass normal checks
heuristics:
   # Do you want to enable AAC's heuristic detection
   enabled: true

   # Heuristic confidence required for a violation. This is calculated as a percentage from 0.0% to 100.0%.
   # Represents how likely a player is to be using killaura, based on heuristic analysis.
   # Must be: 50 <= min_confidence < 100
   # For reference: min_confidence of 85 has been tested to have practically perfect accuracy, but may miss some hacks.
   min_confidence: 75

   # List of patterns which should not incur a violation
   # Should be a 4 character string, usually "P/" followed by two digits
   exempt:
    - "P/00" # This is an example entry. Pattern P/00 does not exist.

   # Map of violation levels to commands.
   # The heuristics check is different to other checks: higher violation level does not mean a higher chance of hacking
   # All violation levels are equal. The default threshold of 10 creates the effect of a delayed ban.
   thresholds:
      1: "aacstaffnotify MINOR: {player} is suspected for Killaura (Heuristic analysis) (ping = {ping}, tps = {tps})"
      2: "aacstaffnotify MODERATE: {player} is suspected for Killaura (Heuristic analysis) (ping = {ping}, tps = {tps})"
      2: "aacstaffnotify MAJOR: {player} is suspected for Killaura (Heuristic analysis) (ping = {ping}, tps = {tps})"

# NoFall ensures players take fall damage
nofall:
   # Do you want to enable the NoFall check
   enabled: true

   # Minimum downwards velocity for NoFall to apply
   required_velocity: -0.2

   # Should AAC deal fall damage to players?
   deal_damage: true
   
   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for NoFall (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MODERATE: {player} is suspected for NoFall (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MAJOR {player} is suspected for NoFall (ping = {ping}, tps = {tps})"

# Jesus stops people from walking on water
jesus:
   # Do you want to enable the Jesus check
   # Disabled by default - functionality covered by Speed/Fly
   enabled: false

   # Should Jesus teleport players?
   teleport: true

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MODERATE: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"
      50: "aacstaffnotify MAJOR: {player} is suspected for Jesus (ping = {ping}, tps = {tps})"

# Phase stops people glitching through blocks
phase:
   # Do you want to enable the Phase check
   enabled: true

   # Which materials should be exempt from phase checking?
   # Items are included in this list due to potenital ViaVersion/cross version compatibility problems with 1.8
   # Should be entered exactly as seen on the Spigot Material enum.
   exclude:
   # Incorrect hitboxes in 1.8 servers
    - "ANVIL"
    - "CHEST"
    - "ENDER_CHEST"
    - "TRAPPED_CHEST"
    - "SOIL"
   # Entity hitboxes
    - "AIR"
    - "STATIONARY_WATER"
    - "WATER"
    - "STATIONARY_LAVA"
    - "LAVA"
   # Protocol hack hitboxes
    - "WATER_LILY"
   # Moving pistons
    - "PISTON_MOVING_PIECE"

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected forPhase (ping = {ping}, tps = {tps})"
      30: "aacstaffnotify MODERATE: {player} is suspected for Phase (ping = {ping}, tps = {tps})"
      50: "aacstaffnotify MAJOR: {player} is suspected for Phase (ping = {ping}, tps = {tps})"

# FastUse stops players consuming items too quickly
fastuse:
   # Do you want to enable the FastUse check
   enabled: true

   # What delay should AAC enforce for eating an item (ms)
   ms_to_use: 1200

   # Map of violation levels to commands
   thresholds:
      2: "aacstaffnotify MINOR: {player} is suspected for FastUse (ping = {ping}, tps = {tps})"
      4: "aacstaffnotify MODERATE: {player} is suspected for FastUse (ping = {ping}, tps = {tps})"
      6: "aacstaffnotify MAJOR: {player} is suspected for FastUse (ping = {ping}, tps = {tps})"

# FastBreak stops players from breaking blocks too quickly
fastbreak:
   # Do you want to enable the fastbreak check?
   enabled: true

   # How many ticks should fastbreak count as lag, and ignore
   let_ticks: 2

   # Should AAC cancel an illegal block break?
   cancel_break: true

   # Should AAC check for a break delay (i.e. between block breaks)
   check_delay: true

   # How many violations before AAC starts cancelling block breaks?
   cancel_vl: 1

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastBreak (ping = {ping}, tps = {tps})"

# FastPlace stops players from placing blocks too quickly
fastplace:
   # Do you want to enable the fastplace check?
   enabled: true

   # How many blocks should a player be allowed to place in one second?
   max_place_per_second: 10

   # Should AAC cancel an illegal block place?
   cancel_place: true

   # How many violations before AAC starts cancelling block placements?
   cancel_vl: 2

   # Map of violation levels to commands
   thresholds:
      10: "aacstaffnotify MINOR: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MODERATE: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"
      20: "aacstaffnotify MAJOR: {player} is suspected for FastPlace (ping = {ping}, tps = {tps})"

# Nuker stops players from breaking too many blocks at once
nuker:
   # Do you want to enable the nuker check?
   enabled: true

   # How many blocks should a player be able to break in 1 second: vl if greater than this value
   max_breaks_per_second: 40

   # The violation will only be recorded if it divides by this number (reduces spam)
   vl_divisor: 41

   # Map of violation levels to commands
   thresholds:
      5: "aacstaffnotify MINOR: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"
      10: "aacstaffnotify MODERATE: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"
      15: "aacstaffnotify MAJOR: {player} is suspected for Nuker (ping = {ping}, tps = {tps})"

# Interact stops players interacting in impossible situations
interact:
   # Do you want to enable the impossibleinteract check?
   enabled: true

   # Detect interacting with liquids?
   cancel_liquids: true

   # Detect ghosthand to interact with blocks?
   cancel_ghost_block: true

   # Detect interacting whilst dead?
   cancel_dead: true

   # Detect hitting whilst using an item?
   cancel_item_attack: true

   # Detect hitting through walls?
   cancel_ghost_attack: true

   # Violation before AAC starts cancelling interact when detecting ghosthand
   cancel_vl_interact: 5
   cancel_vl_place: 5
   cancel_vl_break: 0

   # Map of violation levels to commands
   thresholds:
      15: "aacstaffnotify MINOR: {player} is suspected for ImpossibleInteract (Maybe GhostHand) (ping = {ping}, tps = {tps})"
      25: "aacstaffnotify MODERATE: {player} is suspected for ImpossibleInteract (Maybe GhostHand) (ping = {ping}, tps = {tps})"
      35: "aacstaffnotify MAJOR: {player} is suspected for ImpossibleInteract (Maybe GhostHand) (ping = {ping}, tps = {tps})"

# Velocity makes sure that players take proper knockback
velocity:
   # Do you want to enable the velocity check
   enabled: true

   # How many ticks should AAC check a player for? Higher values compensate for higher ping.
   check_length: 15

   # What is the maximum distance AAC should teleport a detected player?
   # If this is zero, the detected player will not be teleported at all
   max_tp_dist: 0

   # How many violations before AAC starts teleporting players who ignore velocity?
   min_vl: 5

   # How much should AAC increase a player's violation level by on detection?
   vl_normal: 2

   # If a player seems frozen (e.g. changing resource packs/lagging), how much should AAC increase their violation level?
   # 0 by default to prevent false positives with changing resource packs
   vl_no_packet: 0

   # Map of violation levels to commands
   thresholds:
      6: "aacstaffnotify MINOR: {player} is suspected for Velocity (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MODERATE: {player} is suspected for Velocity (ping = {ping}, tps = {tps})"
      18: "aacstaffnotify MAJOR: {player} is suspected for Velocity (ping = {ping}, tps = {tps})"

# Spam prevents players from spamming the server chat
spam:
   # Do you want to enable the spam check?
   enabled: true
   
   # What words should not be covered by the anti-spam check?
   # messages less than 3 letters are already ignored
   whitelist:
     - "lol"
     - "hello"
     - "okay"
   
   # How many messages should a player be able to send in 10 seconds
   message_limit: 4
   
   # How fast can a player send messages (ticks)
   chat_speed: 10
   
   # Map of violation levels to commands
   thresholds:
      6: "aacstaffnotify MINOR: {player} is probably Spamming (ping = {ping}, tps = {tps})"
      12: "aacstaffnotify MODERATE: {player} is probably Spamming (ping = {ping}, tps = {tps})"
      18: "aacstaffnotify MAJOR: {player} is probably Spamming (ping = {ping}, tps = {tps})"

# Parameters for AAC's packet level movement check system. It is required for all non-combat checks
# Monitors movement rate and illegal movement positions.
move:
   # This must be enabled to use any movement check in AAC, and is also required for some world and combat checks.
   # Do not disable unless you are exclusively using combat checks in AAC.
   enabled: true

   # Maximum lagspike size to compensate for (ticks)
   min_balance: -50

   # Maximum extra packets before cancelling (and causing movement violations)
   cancel_threshold: 5

   # Default balance when networking heuristics determine that the connection is normal
   reset_level: -1

   # Maximum packet overload level to hold a player at.
   balance_cap: 10

   # At what tick interval should AAC reduce 1 VL from the player?
   decay: 25

   # Map of violation levels to commands
   thresholds:
      200: "aacstaffnotify MINOR: {player} is suspected for InvalidPackets (Maybe Movement Hack) (ping = {ping}, tps = {tps})"
      300: "aacstaffnotify MODERATE: {player} is suspected for InvalidPackets (Maybe Movement Hack) (ping = {ping}, tps = {tps})"
      400: "aacstaffnotify MAJOR: {player} is suspected for InvalidPackets (Maybe Movement Hack) (ping = {ping}, tps = {tps})"

### General things

# How many times must a player be kicked before he is banned (-1 for no autoban)
# The kick command used must be /aackick for this to work.
# Disabled by default.
ban_threshold: -1

# What command should be executed when a player is banned (only applies for /aackick)
ban_command: "null"

# How long after a player is kicked should he be allowed to log in (ticks)
kick_ticks: 120

# Should AAC kill combat loggers
kill_loggers: false

# Should AAC log events to a file?
log_file: true

# Should /aackick broadcast a message?
aackick_broadcast: false

# Should AAC disable/enable checks dynamically? IF YOU HAVE PWP DISABLE
usetps: true

# Enable verbose logging? This splits log output and pipes some to administrators. Not recommended for normal use.`

**Comment by CeIebrimbor** [comment]
> After updating, it has become much worse.

Your original issue was "AAC Is Kicking And Broadcasting And I'm Sure I Set It Not To? How Do I Remove Them?". But it sounds like you're switching tracks...so is it really much worse, or is this issue just general complaints now?



> Teleports all my donors and staff to the floor when they are flying

Sounds like your fly given to staff is not compatible. AAC really isn't setup to detect movement hacks on Creative or servers that give out fly. Consider giving Staff members the AAC.bypass permission.



> Still sends kick messages, http://prntscr.com/ghgyfm, step isn't even in the config.

This should not be the case, but maybe it carried over from an old config? Running /aacreload again or restarting should make sure the current config is applied. Take a look at your console when doing this to make sure you don't have any errors.

#General Note
If you have individual issues when it comes to severe false detections, please open up a new issue with the relevant information. Do not just keep posting one issue after another in the same issue report. For general help, it is better to utilize the [AAC Discord](https://discord.gg/uc9w2nx), since your issues now seem to be tied to an inability to tinker with the config.
